### ISSUE TITLE
Prevent potential XSS by default

### ISSUE THREAD

**Issue by vincentjames501** [yogthos/markdown-clj#188]
Example markdown:

```
[click me](javascript:window.onerror=alert;throw%20document.URL)
```

Markdown clj will render:

<img width="1072" alt="image" src="https://user-images.githubusercontent.com/396866/195430842-5a675c50-d4da-4709-a393-d9ffdac792a0.png">

Maybe we force folks to specify specific protocols they want to support and validate the urls? We just discovered this and haven't done much analysis at this point.

**Comment by yogthos** [comment]
Yeah, that looks like it would need additional sanitizing to be done. Any chance you'd have a chance to take a look at adding escaping for this?
